Gruntfile.js

```diff
@@ -735,34 +728,50 @@ module.exports = function (grunt) {
 
   grunt.loadNpmTasks('grunt-angular-templates');
 
+  // ex:
+  // grunt test-unit
+  // grunt test-unit --skipRebuild (to save time between test runs)
   // karma must run prior to coverage since karma will generate the coverage results
-  grunt.registerTask('test-unit', [
-    'clean:server',
-    'concurrent:test',
-    'postcss',
-    'connect:test',
-    'karma'
-    // 'coverage' - add back if we want to enforce coverage percentages
-  ]);
+  grunt.registerTask('test-unit',
+    grunt.option('skipRebuild') ?
+      [
+        'connect:test',
+        'karma'
+      ] :
+      [
+        'clean:server',
+        'concurrent:test',
+        'postcss',
+        'connect:test',
+        'karma'
+        // 'coverage' - add back if we want to enforce coverage percentages
+      ]
+    );
 
-  // test as an alias to unit.  after updating protractor,
-  // will make test an alias for both unit & e2e
+  // alias
+  // TODO: 'test' should run both unit & e2e tests
   grunt.registerTask('test', ['test-unit']);
 
+  // ex:
+  // grunt test-integration
+  // grunt test-integration --baseUrl=      (for build environments)
```

Comment:
I'm unclear what "for build environments" means.

Reply:
I'm not 100% sure either, its a rewording of previous comment:

```
// if a baseUrl is defined assume we dont want to run the local grunt server
```
I have no idea why this was included.  Perhaps we can drop the line && the flag.